bors.toml

```diff
@@ -29,4 +29,4 @@ status = [
   "Documentation",
 ]
 
-timeout_sec = 3600
```

Comment:
Why did you bump the timeout?

Reply:
If all of the Buildkite runners are in use, Bors might need to wait a while. This doesn't count towards the Buildkite timeout - once the Buildkite job starts, it has a timeout of 60 minutes. But the time waiting for a Buildkite runner to become available does count towards the Bors timeout. So if all the Buildkite runners are in use, we might hit the Bors timeout of 60 minutes. So I increased it to 180 minutes.

We can decrease it again if you want.